api/src/pcapi/scripts/transform_virtual_venues/main.py

```diff
@@ -0,0 +1,152 @@
+"""
+Script to transform virtual venues into physical ones, meaning:
+
+    1. the venue is not marked virtual (isVirtual) anymore
+    2. its code changes
+    3. it must have a siret (should its offerer head quarters)
+    4. it must create and offerer address link
+"""
+
+import logging
+from typing import Collection
+
+from flask_sqlalchemy import BaseQuery
+import sqlalchemy.orm as sa_orm
+
+import pcapi.connectors.entreprise.api as entreprise_api
+from pcapi.connectors.entreprise.models import SiretInfo
+from pcapi.core.geography import repository as geography_repository
+from pcapi.core.geography.models import Address
+from pcapi.core.offerers.models import OffererAddress
+from pcapi.core.offerers.models import Venue
+from pcapi.core.offerers.schemas import VenueTypeCode
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+QUERY = """
+select 
+    venue.id
+from
+    venue
+left join offerer on offerer.id = venue."managingOffererId"
```

Comment:
La jointure n'apporte rien il me semble.
```suggestion
select 
    venue.id
from
    venue
```

Reply:
J'ai repris la requête SQL du ticket, je pensais d'ailleurs que ça venait de toi =P